static/app/components/replays/playerDOMAlert.tsx

```diff
@@ -1,26 +1,15 @@
-import {useEffect, useState} from 'react';
 import styled from '@emotion/styled';
 
 import {Button} from 'sentry/components/button';
 import {IconClose, IconInfo} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
-import localStorage from 'sentry/utils/localStorage';
+import useDismissAlert from 'sentry/utils/useDismissAlert';
 
 const LOCAL_STORAGE_KEY = 'replay-player-dom-alert-dismissed';
 
 function PlayerDOMAlert() {
-  const [isDismissed, setIsDismissed] = useState(true);
-
-  const handleDismiss = () => {
-    localStorage.setItem(LOCAL_STORAGE_KEY, '1');
-    setIsDismissed(true);
-  };
-
-  useEffect(() => {
-    const val = localStorage.getItem(LOCAL_STORAGE_KEY);
-    setIsDismissed(val === '1');
-  }, []);
+  const {dismiss, isDismissed} = useDismissAlert({key: LOCAL_STORAGE_KEY});
```

Comment:
hm, shouldn't this be named `onDismiss` to indicate that an event?

Reply:
never mind... I just noticed that React Query names event handlers like this too  🤷‍♀️ https://tanstack.com/query/v4/docs/react/reference/useQuery